sdks/node/usage-guide-messaging.md

```diff
@@ -0,0 +1,57 @@
+# Usage and Migration Guide for Node.js Messaging 3.x
+
+Bandwidth's Node.js Messaging SDK has seen significant changes when moving from 2.x to 3.x. Outlined below are a number of common scenarios where you'll see changes.
+
+---
+
+## Client Initialization
+
+### 3.x
+
+```js
+import { Client, ApiController } from '@bandwidth/messaging';
+
+const client = new Client({
+    basicAuthPassword: 'password',
+    basicAuthUserName: 'username'
+});
+
+const controller = new ApiController(client);
+```
+
+### 2.x
+
+```js
+const BandwidthMessaging = require('@bandwidth/messaging');
+BandwidthMessaging.Configuration.basicAuthUserName = "username";
+BandwidthMessaging.Configuration.basicAuthPassword = "password";
+const controller = BandwidthMessaging.APIController;
+```
+
+---
+
+## Send Text Message
+
+### 3.x
+
+```js
+const response = await controller.createMessage('user-id-123', {
```

Comment:
user-id is confusing - this should be accountId - customers get tripped up and put their api username in that field

Reply:
Is there a push to update this within the SDK? Currently the SDK references this as `userId`.

Reply:
If not then there should be - that trips up customers all the time